hardware_layer/hardware_arduino/src/MS5837.cpp

```diff
@@ -1,190 +1,214 @@
 #include "MS5837.h"
```

Comment:
this files were to be left untouched :/ thats why I modified the roslint in Cmake to not check these for lint.